tests/umock_c_negt_ut/umock_c_negative_tests_ut.c

```diff
@@ -657,4 +680,100 @@ TEST_FUNCTION(umock_c_negative_tests_call_count_when_the_module_is_not_initializ
     ASSERT_ARE_EQUAL(size_t, 0, umock_c_indicate_error_call_count);
 }
 
+
+TEST_FUNCTION(umock_c_negative_tests_fail_call_calls_the_call_recorder_fail_call_index_0_result_0)
+{
```

Comment:
Missing Tests_

Reply:
We're doing tagging on these repos (umock_c, uamqp, c shared - I hope :-))

---
In reply to: [224578267](https://github.com/Azure/umock-c/pull/44#discussion_r224578267) [](ancestors = 224578267)